core/src/main/scala/org/broadinstitute/dsde/rawls/util/TracingUtils.scala

```diff
@@ -96,6 +96,33 @@ object TracingUtils {
   def traceFuture[T](name: String)(f: RawlsTracingContext => Future[T])(implicit ec: ExecutionContext): Future[T] =
     traceFutureWithParent(name, RawlsTracingContext(otelContext = Option(Context.root())))(f)
 
+  /**
+    * Trace a sync operation with the RawlsRequestContext
+    *
+    * @param name The name that will be used in the trace
+    * @param parentContext the RawlsRequestContext to use for tracing, if `parentContext.otelContext` is defined
+    * @param f the operation to execute within the trace
+    * @return the result of the operation `f`
+    */
+  def trace[T](name: String, parentContext: RawlsRequestContext)(f: RawlsRequestContext => T): T =
```

Comment:
Not having this was pushing a lot of different places to use a `Future` where it wasn't really necessary.

Reply:
This appears to be equivalent to `traceNakedWithParent` below

Reply:
That's using` RawlsTracingContext` not `RawlsRequestContext`.

It's arguable that the better approach would be an overload or something, but we have similar divergence between others, such as between `traceFutureWithParent` and `traceFuture`. 
Also, this is the most common/basic/unembellished instance of this, and therefore worth giving the most basic name. But I didn't feel like renaming usages of `traceNakedWithParent` in what is already a substantial PR.

Reply:
Oh, right. Reading is hard. I prefer this to overloading 👍